crates/net/downloaders/src/lib.rs

```diff
@@ -27,10 +27,29 @@ pub mod metrics;
 /// efficiently buffering headers and bodies for retrieval.
 pub mod file_client;
 
+/// Module managing file-based data retrieval and buffering of receipts.
+///
+/// Contains [ReceiptFileClient](receipt_file_client::ReceiptFileClient) to read receipt data from
+/// files, efficiently buffering receipts for retrieval.
+///
+/// Currently configured to use codec [`HackReceipt`](file_codec_ovm_receipt::HackReceipt) based on
+/// export of below Bedrock data using <https://github.com/testinprod-io/op-geth/pull/1>. Codec can
+/// be replaced with regular encoding of receipts for export.
+///
+/// NOTE: receipts can be exported using regular op-geth encoding for `Receipt` type, to fit
+/// reth's needs for importing. However, this would require patching the diff in <https://github.com/testinprod-io/op-geth/pull/1> to export the `Receipt` and not `HackReceipt` type (originally
+/// made for op-erigon's import needs).
+pub mod receipt_file_client;
```

Comment:
making a mental note here that we'll move all of this to OP modules eventually

Reply:
with this in mind I kept everything op specific in the codec. no need to move the receipts file client, but it needs to be made generic over the codec, which is rn set to `HackReceiptCodec`. will open an issue for this.

Reply:
sounds good!